src/adapters/poi/idunn_poi.js

```diff
@@ -58,6 +58,9 @@ export default class IdunnPoi extends Poi {
       if(err === 404) {
         return
       }
+      else if(err === 400){
```

Comment:
Why not combining this condition with the previous one considering there is no specific treatment?